.github/workflows/tests.yml

```diff
@@ -33,5 +33,5 @@ jobs:
       #     docker network prune -f || true
 
       # run tests
-      - name: run tests
-        run: go test -timeout 30m -v -p 2 ./...
+      - name: run all tests
+        run: (go test -timeout 30m -v -p 2 ./...) || (echo "\n\n*****CHAIN and RELAYER LOGS*****" && tail -n 20 "$HOME/.ibctest/logs/ibctest.log" && exit 1)
```

Comment:
Looks like you are purposefully throwing an error in the log tail so that it won't consider that for pass/fail, only the go test exit code will be considered? Fine by me, just took a minute to understand the intention.

Reply:
I'll add a comment. Basically, show the chain and relayer logs on a test failure.